licenses/views.py

```diff
@@ -100,18 +111,37 @@ def view_license(request, license_code, version, jurisdiction=None, language_cod
         legalcode.license.legal_codes.all(), language_code, "license"
     )
 
+    kwargs = dict(
+        template_name="legalcode_page.html",
+        context={
+            "fat_code": legalcode.license.fat_code(),
+            "languages_and_links": languages_and_links,
+            "link_to_plain_text_file": f"{ legalcode.license_url() }/index.txt"
+            if legalcode.license_url().endswith("legalcode")
+            else f"{ legalcode.license_url() }.txt",
```

Comment:
Could we just define a `plaintext_url()` method on the LegalCode class that returns the URL we need, like the `license_url()` and `deed_url()` methods?

Reply:
done